extensions/ql-vscode/src/view/result-tables.tsx

```diff
@@ -144,7 +151,7 @@ export class ResultTables
           resultsPath={this.props.resultsPath}
           sortState={this.props.sortStates.get(resultSet.schema.name)} />
       }
-    </div>;
+    </div >;
```

Comment:
Super nit: remove this space for consistency.  I'll file an issue to set up a code formatting linter.

Reply:
Done. I don't know why `tsfmt` is a no-op on jsx syntax sometimes. I feel like it's sufficient for most other formatting concerns.